core/core.go

```diff
@@ -42,6 +45,9 @@ import (
 )
 
 const VERSION = "0.0.1"
+const apiURL = "https://api.textile.io/"
```

Comment:
i think the extra slash is duplicate in its current usage below, other than that, hell yeah